models/ed/R/met2model.ED2.R

```diff
@@ -137,8 +148,9 @@ met2model.ED2 <- function(in.path, in.prefix, outfolder, start_date, end_date, l
     slen <- seq_along(SW)
     Tair <- c(rep(Tair[1], toff), Tair)[slen]
     Qair <- c(rep(Qair[1], toff), Qair)[slen]
-    U    <- c(rep(U[1], toff), U)[slen]
-    V    <- c(rep(V[1], toff), V)[slen]
+    # U  <- c(rep(U[1], toff), U)[slen]
```

Comment:
just remove this code that is commented out, we always have git history to look at why it is not here. Also the comment above is fine.

Reply:
thanks for pointing this out, I commented them out instead of deleting as a reminder, I was meaning to ask @mdietze whether this was how he wanted me to implement met2model changes for wind

Reply:
per above, this is not how I wanted this implemented. Let's revert to the previous code